content/aws/iam/assuming-roles.md

```diff
@@ -7,3 +7,32 @@ tags:
   - aws
   - assume-role
 ---
+
+# Assume Role via AWS Web Console
+
+**NOTE**: Due to the security implications, IAM policies are set up to **only** allow the root AWS account to assume roles into other accounts.
+
+1. Log into the AWS root acccount
+{{< img src="/assets/aws-root-login.png" title="Example AWS root login" >}}
+
+2. Click on `user@example.com @ example-root-aws` drop down at the top of the console and select `Switch Role`
+  - Enter the AWS Account ID of the account to assume in `Account` field
+  - Use `OrganizationAccountAccessRole` as the `Role`
+  - (Optional) Pick `Display Name` and choose a `Color` for the role
+
+{{< img src="/assets/aws-switch-role.png" title="Example AWS switch role" >}}
+
+# Assume Role via CLI (using aws-vault)
+
+First, ensure that the proper profiles are setup following [Authorization]({{< relref "aws/iam/authorization.md" >}}).
+
+
+## Inspect the environment
+```
+$ aws-vault exec `<profile>` -- env | grep AWS
```

Comment:
this is not right. We shouldn't use the ticks inside of a codeblock because they have a signficant meaning in bash. 

My preference is to use env-style notation (e.g. `$profile`). This is not inventing a new convention and is valid shell code.